### PR TITLE
feat: Disable the Apollo Client cache

### PR DESCRIPTION
**Pull request by gorandalum**
The theory is that the Apollo Client cache is the reason why we are
seeing a large memory usage in prod. Also the client doesn't make as
much sense when we have multiple BFF instances running in prod.

The main usage for the Apollo Client cache was to cache the previous
result for realtime updates, so only calls that were actually
updated could be returned. This is now changed to only return
updates for calls up to 30 minutes in the future, and no usage of
cache.

Here are some results of testing the realtime endpoint with old and new BFF where the new BFF does not use Apollo client cache, but instead restrict realtime updates to the next 30 minutes. 

The test was done around 10:30 AM a normal weekday.

| Case | Old BFF | New BFF |
| --- | --- | --- |
| Departures screen: P1 with date now | ~9 updates | ~20 updates |
| Front page: Multiple favorites | ~7 updates | ~15 updates | 

In summary it seems that the new BFF returns around double the updates, but luckily still fairly low numbers so it shouldn't be a too high increase for the user with regards to mobile data usage.

### CODE REVIEW

**Review by rosvik — APPROVED** [review]
LGTM 👍
